demo/fake_gan_2/run.py

```diff
@@ -10,14 +10,15 @@
 def fake_gan(desc):
     if desc == "NSFW":
         raise gr.Error("NSFW - banned content.")
-    time.sleep(5)
+    if desc == "":
+        raise ValueError("Too short")
+    time.sleep(12)
```

Comment:
I would remove this. Good for testing, not great for a demo
```suggestion
    time.sleep(12)
```